mutation-testing/output-pr-mutants/action.yml

```diff
@@ -0,0 +1,166 @@
+              echo "</details>" >> "$GITHUB_STEP_SUMMARY"
+              echo "" >> "$GITHUB_STEP_SUMMARY"
+            fi
+
+            if [[ "$section_title" != "" ]]; then
+              awk -F':' '{printf "- [ ] " "[" $0 "]"; file_path=$1; line=$2; $1=""; $2=""; printf "(" "'"$server_url"'/'"$organisation"'/'"$repository"'/blob/'"$commit"'/" file_path "#L" line-1 ")\n\n"}' "$file_name" >> "$GITHUB_STEP_SUMMARY"
+            else
+              awk -F':' '{printf "- [x] " "[" $0 "]"; file_path=$1; line=$2; $1=""; $2=""; printf "(" "'"$server_url"'/'"$organisation"'/'"$repository"'/blob/'"$commit"'/" file_path "#L" line-1 ")\n\n"}' "$file_name" >> "$GITHUB_STEP_SUMMARY"
+            fi
+
+            if [[ "$section_title" == "Missed:" ]]; then
+              echo "### To resolve this issue, consider one of the following options:" >> "$GITHUB_STEP_SUMMARY"
+              echo "- Modify or add tests including this function." >> "$GITHUB_STEP_SUMMARY"
+              echo "- If you are absolutely certain that this function should not undergo mutation testing, add '#[mutants::skip]' or '#[cfg_attr(test, mutants::skip)]' function header to skip it." >> "$GITHUB_STEP_SUMMARY"
+            elif [[ "$section_title" == "Timeout:" ]]; then
+              echo "### To resolve this issue, consider one of the following options:" >> "$GITHUB_STEP_SUMMARY"
+              echo "- Modify the tests that include this funcion." >> "$GITHUB_STEP_SUMMARY"
+              echo "- Add '#[mutants::skip]' or '#[cfg_attr(test, mutants::skip)]' function header to skip it." >> "$GITHUB_STEP_SUMMARY"
+            elif [[ "$section_title" == "Unviable:" ]]; then
+              echo "### To resolve this issue, consider one of the following options:" >> "$GITHUB_STEP_SUMMARY"
+              echo "- Create 'Default::default()' implementation for the specific structure." >> "$GITHUB_STEP_SUMMARY"
+              echo "- Add '#[mutants::skip]' or '#[cfg_attr(test, mutants::skip)]' function header to skip it." >> "$GITHUB_STEP_SUMMARY"
+            fi
+
+            echo >> "$GITHUB_STEP_SUMMARY"
+          fi
+        }
+
+        # Print uncaught (missed/timeout/unviable) mutants to summary
+        echo "# Uncaught Mutants" >> "$GITHUB_STEP_SUMMARY"
+        write_section "Missed:" "./mutants-shards/missed.txt"
+        write_section "Timeout:" "./mutants-shards/timeout.txt"
+        write_section "Unviable:" "./mutants-shards/unviable.txt"
+
+        # Print caught mutants to summary
+        echo "# Caught Mutants" >> "$GITHUB_STEP_SUMMARY"
+        write_section "" "./mutants-shards/caught.txt"
+
+        # Get most relevant exit code from the file and match it
+        exit_code=$(<"mutants-shards/exit_code.txt")
+
+        case $exit_code in
+          0)
+              if [[ -f ./mutants-shards/unviable.txt ]]; then
+                echo "Found unviable mutants!"
```

Comment:
could this be sent to GITHUB_OUTPUT, or is the amount of data too large potentially?
this could be useful for debugging

Reply:
Do you mean by `send to GITHUB_OUTPUT` to send the exit code, the message, or the mutants from the file? If you're talking about the exit code or log message, they are small and can be added. If it's about the mutant outcomes from the file, they are already printed to step summary.